sys/include/fido2/ctap/ctap_mem.h

```diff
@@ -43,7 +43,12 @@ extern "C" {
 /**
  * @brief   Default amount of flashpages to use
  */
-#define CTAP_FLASH_DEFAULT_AMOUNT_FLASHPAGES 4
+#if defined(CONFIG_FIDO2_CTAP_NUM_FLASHPAGES) && \
+    (CONFIG_FIDO2_CTAP_NUM_FLASHPAGES >= 2)
+#define CTAP_FLASH_NUM_FLASHPAGES CONFIG_FIDO2_CTAP_NUM_FLASHPAGES
+#else
+#define CTAP_FLASH_NUM_FLASHPAGES 4
+#endif
```

Comment:
How about:

```
#ifndef CONFIG_FIDO2_CTAP_NUM_FLASHPAGES
#define CONFIG_FIDO2_CTAP_NUM_FLASHPAGES 4
#endif

#if CONFIG_FIDO2_CTAP_NUM_FLASHPAGES <= 2 || CONFIG_FIDO2_CTAP_NUM_FLASHPAGES >= 256
#error "why failure?"
#endif
```
(haven't tested it)

This way you would get rid of the extra symbol.

Reply:
This would also work. I would change `CONFIG_FIDO2_CTAP_NUM_FLASHPAGES >= 256` to `CONFIG_FIDO2_CTAP_NUM_FLASHPAGES >= FLASHPAGE_NUMOF` though.

Overall it is hard to give an upper bound for the max number of flashpages CTAP can obtain ...

Reply:
Sounds good, please go ahead